cypress/e2e/user-flows/scenario_008.cy.js

```diff
@@ -0,0 +1,109 @@
+import { SimulatorPage } from "../../pages/simulator/simulatorPage";
+import { EditorPage } from "../../pages/simulator/editorPage";
+import { SlicePage } from "../../pages/slices/slicePage";
+import { SLICE_MOCK_FILE } from "../../consts";
+
+const SLICE = {
+  id: "scenario008",
+  name: "Scenario008",
+  library: "slices",
+};
+
+describe("Scenario 008", () => {
+  let slice = new SlicePage();
+  let simulator = new SimulatorPage();
+  let editor = new EditorPage();
+
+  beforeEach(() => {
+    cy.clearProject();
+    cy.setSliceMachineUserContext({});
+  });
+
+  it("creates slice and tests the editor mocks", () => {
```

Comment:
are we sure we want one IT only ? or one per action in the scenario ?

Reply:
The name of the action should be clearer.
What exactly are you testing ? the display ? a specific screen or functionality ?

Reply:
The problem separating in multiple `it` sections is that the page has to be reloaded between each block, which is quite slow currently, and not what the user would do. Open for discussion though

Reply:
I opened the discussion because i'm not clear on what we want to do for this actually 😅 
I think @mdeclercq  would be able to answer what's the best practice for us

Reply:
I like the idea of splitting the scenario into multiple steps to make it easier to understand but I don't think it's the right approach for e2e testing with Cypress.
They actually discourage having multiple `it()` tests relying on each other, see https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests
@mathieud, good point on page reload slowness, I experienced that too :)